### PR TITLE
Migrate icons to codicon in dependency explorer

### LINKED ISSUE #246: Migrate to VS Code Icons
We plan to migrate all of the icons in dependency explorer to [VS Code Icons](https://microsoft.github.io/vscode-codicons/dist/codicon.html).

This is for the purpose to better support different themes in VS Code. If the theme creator designs the specific icon, them the icon will change if the theme changes.

Below is the new looking of the explorer:
explorer

### PR DESCRIPTION
**Pull request by jdneo**
resolve #246 

This PR is to totally migrate the icons in dependency explorer to `codicon`.

Before:
Screen Shot 2020-03-13 at 10 59 39 AM
After:
Screen Shot 2020-03-13 at 11 18 15 AM

@testforstephen @akaroml @Eskibear Please take a look when you have time, let me know if this UI change is acceptable or if you have any suggestions.

_Reference: https://microsoft.github.io/vscode-codicons/dist/codicon.html_

### CODE REVIEW

File: src/views/containerNode.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export class ContainerNode extends DataNode {
         return `container/${this.name}`;
     }
 
-    protected get iconPath(): { light: string, dark: string } {
-        return ExplorerNode.resolveIconPath("library");
+    protected get iconPath(): ThemeIcon {
+        return new ThemeIcon("inbox");
```
Comment (akaroml) [inline src/views/containerNode.ts]: We are actually looking for "books" right? But it's not there. How about " files"?

File: src/views/packageNode.ts [hunk 2]
```diff
@@ -39,7 +40,7 @@ export class PackageNode extends DataNode {
         return result;
     }
 
-    protected get iconPath(): { light: string; dark: string } {
-        return ExplorerNode.resolveIconPath("package");
+    protected get iconPath(): ThemeIcon {
+        return new ThemeIcon("symbol-namespace");
```
Comment (akaroml) [inline src/views/packageNode.ts]: I would suggest "symbol-package".

File: src/views/packageRootNode.ts [hunk 3]
```diff
@@ -63,12 +64,12 @@ export class PackageRootNode extends DataNode {
         }
     }
 
-    protected get iconPath(): { light: string; dark: string } {
+    protected get iconPath(): ThemeIcon {
         const data = <IPackageRootNodeData>this.nodeData;
         if (data.entryKind === PackageRootKind.K_BINARY) {
-            return ExplorerNode.resolveIconPath("jar");
+            return new ThemeIcon("archive");
```
Comment (akaroml) [inline src/views/packageRootNode.ts]: Suggest "file-zip"

File: src/views/containerNode.ts (hunk 1 above)
Comment (akaroml) [reply src/views/containerNode.ts]: Can we request books in vscode repo?

**Comment by jdneo** [comment]
Now:
Screen Shot 2020-03-13 at 5 55 36 PM

File: src/views/containerNode.ts (hunk 1 above)
Comment (jdneo) [reply src/views/containerNode.ts]: Requested: https://github.com/microsoft/vscode-codicons/issues/7

**Pull request merged**